CalmspaceTests/CalmspaceTests.swift

```diff
@@ -3,52 +3,64 @@
 //  CalmspaceTests
 //
 //  Created by Admin on 16/07/2023.
-//
 
 import XCTest
 @testable import Calmspace
 
 final class CalmspaceTests: XCTestCase {
     
-    var viewController: FirstAskViewController!
+    var viewController: AdviceViewController!
     
-    override func setUpWithError() throws {
+    override func setUp() {
         super.setUp()
-        
         let storyboard = UIStoryboard(name: "Main", bundle: nil)
-        guard let viewController = storyboard.instantiateViewController(withIdentifier: "FirstAskViewController") as? FirstAskViewController else {
-            XCTFail("Failed to instantiate FirstAskViewController from storyboard")
-            return
-        }
-        self.viewController = viewController
-        
-        print("FirstAskViewController instantiated successfully")
-        viewController.loadViewIfNeeded()
-        print("View loaded successfully")
+        viewController = storyboard.instantiateViewController(withIdentifier: "AdviceViewController") as? AdviceViewController
+        _ = viewController.view
     }
     
-    override func tearDownWithError() throws {
+    override func tearDown() {
         viewController = nil
         super.tearDown()
     }
     
-    func testUIElementsExist() {
-        XCTAssertNotNil(viewController.questionText, "questionText outlet is not set")
-        XCTAssertNotNil(viewController.answerButton1, "answerButton1 outlet is not set")
-        XCTAssertNotNil(viewController.answerButton2, "answerButton2 outlet is not set")
-        XCTAssertNotNil(viewController.answerButton3, "answerButton3 outlet is not set")
-        XCTAssertNotNil(viewController.backgroundImageView, "backgroundImageView outlet is not set")
-    }
-
-    func testButtonFontIsSet() {
-        let button = UIButton()
-        viewController.setButtonFont(for: button)
-        XCTAssertEqual(button.titleLabel?.font, UIFont(name: "AvenirNext-DemiBold", size: 19))
+    func testAdviceSelection() {
```

Comment:
This one looks more like a view test than a unit test. Unit tests don't normally do anything with view controllers.